apps/web-giddh/src/app/new-inventory/component/inventory-dashboard/dashboard-profit-stocklist/dashboard-profit-stocklist.component.ts

```diff
@@ -0,0 +1,92 @@
+import { Component, OnInit, ViewChild, ElementRef} from '@angular/core';
+import { GeneralService } from '../../../../services/general.service';
+import { BsModalRef, BsModalService } from 'ngx-bootstrap/modal';
+import { GIDDH_DATE_RANGE_PICKER_RANGES } from '../../../../app.constant';
+import * as moment from 'moment/moment';
+import { GIDDH_DATE_FORMAT, GIDDH_NEW_DATE_FORMAT_UI } from '../../../../shared/helpers/defaultDateFormat';
+import { BreakpointObserver } from '@angular/cdk/layout';
+
+@Component({
+    selector: 'dashboard-profit-stocklist',
+    templateUrl: './dashboard-profit-stocklist.component.html',
+    styleUrls: ['./dashboard-profit-stocklist.component.scss'],
+
+})
+
+export class DashboardProfitStockList implements OnInit {
+    /* this will store image path*/
+    public imgPath: string = '';
+    /* This will store modal reference */
+    public modalRef: BsModalRef;
+    /* This will store selected date range to use in api */
+    public selectedDateRange: any;
+    /* This will store selected date range to show on UI */
+    public selectedDateRangeUi: any;
+    /* This will store available date ranges */
+    public datePickerOption: any = GIDDH_DATE_RANGE_PICKER_RANGES;
+    /* Moment object */
+    public moment = moment;
+    /* Selected from date */
+    public fromDate: string;
+    /* Selected to date */
+    public toDate: string;
+    /* Selected range label */
+    public selectedRangeLabel: any = "";
+    /* This will store the x/y position of the field to show datepicker under it */
+    public dateFieldPosition: any = { x: 0, y: 0 };
+
+    @ViewChild('datepickerTemplate') public datepickerTemplate: ElementRef;
+
+    constructor(
+        private generalService: GeneralService, private modalService: BsModalService,
+        private _breakPointObservar: BreakpointObserver,) {
+    }
+    /*datepicker funcation*/
+    public showGiddhDatepicker(element: any): void {
+        if (element) {
+            this.dateFieldPosition = this.generalService.getPosition(element.target);
+        }
+        this.modalRef = this.modalService.show(
+            this.datepickerTemplate,
+            Object.assign({}, { class: 'modal-lg giddh-datepicker-modal', backdrop: false, ignoreBackdropClick: false })
+        );
+    }
+    /**
+     * This will hide the datepicker
+     *
+     * @memberof DaybookComponent
```

Comment:
Update the `@memberof` with current class through out the file
```suggestion
     * @memberof DashboardProfitStockList
```